f2/src/forms/EvidenceInfoForm.js

```diff
@@ -0,0 +1,174 @@
+                                >
+                                    {i18n._(status)}
+                                </PseudoBox>
+                            ) : null}
+                        </Stack>
+
+                        <Container>
+                            {files.map((f, index) => (
+                                <React.Fragment key={index}>
+                                    <InfoCard mb={4}>
+                                        <Stack spacing={4}>
+                                            <H3>{f.name}</H3>
+                                            <Box>
+                                                <Field
+                                                    name={`file-description-${index}`}
+                                                    input={{
+                                                        value: fileDescriptions[index],
+                                                        onChange,
+                                                    }}
+                                                >
+                                                    {props => (
+                                                        <FormControl>
+                                                            <FormLabel htmlFor={`file-description-${index}`}>
+                                                                <Trans id="evidencePage.fileDescription" />
+                                                            </FormLabel>
+                                                            <TextArea
+                                                                id={`file-description-${index}`}
+                                                                name={props.input.name}
+                                                                value={props.input.value}
+                                                                onChange={props.input.onChange}
+                                                            />
+                                                        </FormControl>
+                                                    )}
+                                                </Field>
+                                            </Box>
+
+                                            <Button
+                                                mr="auto"
+                                                variantColor="red"
+                                                type="button"
+                                                onClick={() => removeFile(index)}
+                                            >
+                                                <Trans id="evidencePage.removeFileButton" />
+                                            </Button>
+                                        </Stack>
+                                    </InfoCard>
+                                </React.Fragment>
+                            ))}
+                        </Container>
+
```

Comment:
You should add a `<P>` block with the "Next, you will be asked about your location." here.

Reply:
fixed